pom.xml

```diff
@@ -73,7 +73,8 @@ limitations under the License.
         <hbase.version.1>1.4.10</hbase.version.1>
         <hbase.version.2>2.2.0</hbase.version.2>
         <hbase.version>${hbase.version.1}</hbase.version>
-        <hadoop.version>2.7.4</hadoop.version>
+        <hadoop.version.1>2.7.4</hadoop.version.1>
+        <hadoop.version.2>2.8.5</hadoop.version.2>
```

Comment:
Can we rename these to `hbase1-hadoop.version` & `hbase2-hadoop.version`? To stay consistent with hbase-guava & hbase-protobuf?